src/components/questionStructure/Task/variants/HandwritingTask/HandwritingEditor.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Card, Flex } from '@radix-ui/themes'
+import { MathJax } from 'better-react-mathjax'
+import React from 'react'
+
+import Markdown from '../../../../Markdown'
+import Canvas from './Canvas'
+
+const HandwritingEditor: React.FC<{
+  username: string
+  latex: string
+  onAnswerChange: (value: string) => void
+  questionText: string
+}> = ({ username, latex, onAnswerChange, questionText }) => {
```

Comment:
Extract the props in dedicated interface (+ remark on redundant `questionText`).
Should this be `ProcessedHandwritingEditor`? Asking due to the `latex` bit.

Reply:
Resolved in d235b04